helm/chaos-mesh/values.yaml

```diff
@@ -398,7 +398,7 @@ dnsServer:
   # Name of serviceaccount for chaos-dns-server.
   serviceAccount: chaos-dns-server
   # Image of DNS Server
-  image: pingcap/coredns:v0.2.1
+  image: chaos-mesh/chaos-coredns:v0.2.5
```

Comment:
it should be `ghcr.io/chaos-mesh/chaos-coredns:v0.2.5`

We put our images on github package instead of the docker repository.

Reply:
Got it! Will fix it soon.

Reply:
docker repository -> dockerhub